src/lib/utils/hooks/SynapseAPI/entity/queryKeys.ts

```diff
@@ -0,0 +1,91 @@
+import { QueryClient } from 'react-query'
+import {
+  EntityBundleRequest,
+  EntityChildrenRequest,
+  QueryBundleRequest,
+} from '../../../synapseTypes'
+
+/**
+ * Co-location of react-query keys for queries related to Synapse entity data. Two objectives of this strategy are
+ * - minimize the risk of inappropriate cache key collisions
+ * - simplify clearing the approprate query caches when the entity data is updated
+ *
+ * For more information, see https://tkdodo.eu/blog/leveraging-the-query-function-context
+ */
+export const entityQueryKeys = {
+  /** Query key for all entities */
+  all: [{ objectType: 'entity' }],
```

Comment:
As long as all of our entity-related queries use one of these keys, we can invalidate all entity cache data with `QueryClient.invalidateQueries(entityQueryKeys.all)`